package.json

```diff
@@ -94,5 +96,8 @@
     "*.php": [
       "scripts/linter-new-php"
     ]
+  },
+  "resolutions": {
+    "@babel/preset-env": "7.10.4"
```

Comment:
I think we have to do this until `calypso-build` updates their dependency.

Reply:
Looks like this needs some extra steps, it's not supported by `npm` itself, just `yarn`. [There is a helper for it:](https://github.com/rogeriochaves/npm-force-resolutions)
```
"scripts": {
  "preinstall": "npx npm-force-resolutions"
}
```

Reply:
Ahh, I had that in there but never pushed it because the build started working :)

Reply:
Yeah that scripts generates a right `package-lock.json` but then `npm install` just overwrites it.